recipes/hana/all/conandata.yml

```diff
@@ -0,0 +1,28 @@
+sources:
+  "boost-1.70.0":
+    url: "https://github.com/boostorg/hana/archive/boost-1.70.0.tar.gz"
+    sha256: "c418d9a9f9e64d35500f6dc0fe4a5edc9d436f7f9f21f933c5da679beb0edb74"
+  "boost-1.70.0.beta1":
```

Comment:
I am not sure we need to merge in pre-releases

Reply:
Fixed in bc421dd